config/i18n.json

```diff
@@ -528,14 +529,12 @@
     "ExcludeItem": "Exclude Item",
     "Filter": "Filters",
     "GeneratedBuilds": "Builds",
-    "LimitedCombos": "You have {{combosWithoutCaps, number}} possible stat combinations, but we only looked at {{combos, number}} of them to keep DIM from crashing. As a result, some possibilities are not shown. Lock some items or perks or dismantle some old armor to get under the limit.",
+    "LimitedCombos": "You have {{combosWithoutCaps, number}} possible item combinations, but we only looked at {{combos, number}} of them to keep DIM from crashing. As a result, some possibilities are not shown. Lock some items or perks or dismantle some old armor to get under the limit.",
```

Comment:
Did you really switch it to item combos from stat combos?

Reply:
Not direct item combos but it now considers 3 things for each item. Stats, energy and which seasonal mod slot it contains.